packages/frontend/app/(dashboard)/profile/components/EditProfile.tsx

```diff
@@ -62,8 +60,6 @@ const EditProfile: React.FC = () => {
         })
     }
 
-    const newUser = await mutate() //update the context
-    setUserInfo({ ...userInfo, ...newUser })
```

Comment:
You might not want to delete this setUserInfo since you want to update the global userInfo context to the new information. I would probably just do something like setUserInfo(newProfile) or something. Make sure to manually test it.